service/resource/s3bucketv1/mocks.go

```diff
@@ -0,0 +1,35 @@
+package s3bucketv1
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/iam"
+)
+
+const (
+	// accountIDIndex represents the index in which we can find the account ID in the user ARN
+	// (splitting by colon)
+	accountIDIndex  = 4
+	accountIDLength = 12
+)
+
+type IAMClientMock struct {
+	accountID string
+}
+
+func (i *IAMClientMock) GetUser(input *iam.GetUserInput) (*iam.GetUserOutput, error) {
```

Comment:
Currently GetUser is duplicated. I think we could move this to an `aws` service. WDYT?